config/environment.rb

```diff
@@ -25,17 +26,7 @@ def self.db_connection
   end
 
   def self.read_env
-    if ENV["SYSOPIA"]
```

Comment:
This method allows to keep environments "in sync" so if someone adds or removes ENV variable in their working file -- it becomes obvious